server/testutils/factories/person.ts

```diff
@@ -9,6 +9,9 @@ export default Factory.define<Person>(() => ({
   name: faker.name.fullName(),
   dateOfBirth: DateFormats.formatApiDate(faker.date.past()),
   sex: faker.helpers.arrayElement(['Male', 'Female', 'Other', 'Prefer not to say']),
+  status: faker.helpers.arrayElement(['InCustody', 'InCommunity']),
+  nomsNumber: `NOMS${faker.datatype.number({ min: 100, max: 999 })}`,
   nationality: faker.address.country(),
-  religion: faker.random.word(),
+  religionOrBelief: faker.helpers.arrayElement(['Christian', 'Muslim', 'Jewish', 'Hindu', 'Buddhist', 'Sikh', 'None']),
+  prisonName: `HMP ${faker.address.street()}`,
```

Comment:
Small point, but in the interests of keeping the faked/fixture data vaguely realistic maybe we ought to link `status` and `prisonName` as it only makes sense to show a prison name if the person is in custody. 

Again, perhaps hardcoding would be easier?